### ISSUE TITLE
Remove old pages (404s) from Google index

### ISSUE THREAD

**Issue by cappadona** [cul-it/mann-wagon#828]
@cappadona commented on [Tue Jan 31 2017](https://github.com/cul-it/mann-sitefeedback/issues/26)

**Comment by tahirpoduska** [comment]
https://www.google.com/webmasters/tools/removals?pli=1

**Comment by cappadona** [comment]
Thanks...looks pretty similar to the [url-removal](https://www.google.com/webmasters/tools/url-removal) I've been using in the Search Console, which apparently is the [preferable method since we own the domain](https://support.google.com/webmasters/answer/7041154).

As long as we actually remove the content or block Googlebot for content that remains (not applicable here), the removal request should hold even after it automatically expires, because the content will not be re-indexed.

I think I've addressed our most egregious 404 URLs by scouring through the top 50 landing pages since we launched as well as the top 25 404s through a custom analytics report, which we can continue to monitor. For some of these I've submitted requests for removal, for others I've setup redirects within the back office.
